lib/super_good/solidus_taxjar/reporting.rb

```diff
@@ -6,6 +6,7 @@ def initialize(api: SuperGood::SolidusTaxjar.api)
       end
 
       def report_transaction(order)
+        return unless SuperGood::SolidusTaxjar.reporting_enabled
```

Comment:
It would be nice if this was checked inside of the subscriber instead of here. That way people could opt-out of the automated reporting, but still manually call the reporting API. Thoughts on that?